std/sys/thread/Tls.hx

```diff
@@ -28,7 +28,10 @@ package sys.thread;
 
 /**
 	Creates thread local storage.
- */
+
+	(hl) Warning: At the moment `Tls` does not protect the value from being
```

Comment:
That sounds like something we should fix, not document...

Reply:
https://github.com/HaxeFoundation/hashlink/issues/284